build/yaml/samples/nsx_v1alpha1_vpcnetworkconfigurations.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: nsx.vmware.com/v1alpha1
+kind: VPCNetworkConfiguration
+metadata:
+  name: vpc-network-config1
+  namespace: ns-1
+spec:
+  appliedToNamespaces:
+    - ns-1
+  tier0Gateway: /infra/tier-0s/t0
+  edgeCluster: /infra/sites/default/enforcement-points/default/edge-clusters/2d9df59f-6dc6-4911-8865-21fadc23d4da
+  defaultSubnetPrefixLength: 28
+  nsxtProject: proj-1
+  publicIPv4Block:
+    - /infra/ip-blocks/block1
+  privateCIDRs:
+    - 172.26.0.0/16
+    - 172.36.0.0/16
+  ingressCIDRs:
```

Comment:
Do we still have `ingressCIDRs`?

Reply:
sorry， forgot to remove it. Removed now.